values.yaml

```
@@ -448,6 +454,7 @@ rabbitmq-ha:
     # Enable Queue Mirroring between nodes
     # See https://www.rabbitmq.com/ha.html
     policies: '{"vhost":"/","name":"ha","pattern":"", "definition":{"ha-mode":"all","ha-sync-mode":"automatic","ha-sync-batch-size":10}}'
+  resources: {}
```

Taken from https://github.com/helm/charts/blob/609560713f68f6f88cc5e7ae744d306a555a1681/stable/rabbitmq-ha/values.yaml#L356-L379

```suggestion
  # We recommend to set the memory limit for RabbitMQ-HA Pods in production deployments.
  # Make sure to also change the rabbitmqMemoryHighWatermark following the formula:
  # rabbitmqMemoryHighWatermark = 0.4 * resources.limits.memory
  resources: {}
```

---

Agree!